builder/dockerfile/builder.go

```diff
@@ -148,7 +148,7 @@ func newBuilder(ctx context.Context, options builderOptions) (*Builder, error) {
 		Output:           options.ProgressWriter.Output,
 		docker:           options.Backend,
 		idMapping:        options.IDMapping,
-		imageSources:     newImageSources(ctx, options),
+		imageSources:     newImageSources(options),
```

Comment:
This commit looks like a fix-up for the `builder: Don't store context in struct` commit; probably best to squash with that one.

Reply:
Done